documentation/projects/proposals/document_all_media_properties/20230426-implementation_plan_documenting_media_properties.md

```diff
@@ -0,0 +1,258 @@
+# 2023-04-26 Implementation Plan: Document all media properties in the catalog
+
+<!-- See the implementation plan guide for more information: https://github.com/WordPress/openverse/tree/19791f51c063d0979112f4b9f4eeace04c8cf5ff/docs/projects#implementation-plans-status-in-rfc -->
+<!-- This template is exhaustive and may include sections which aren't relevant to your project. Feel free to remove any sections which would not be useful to have. -->
+
+## Reviewers
+
+<!-- Choose two people at your discretion who make sense to review this based on their existing expertise. Check in to make sure folks aren't currently reviewing more than one other proposal or RFC. -->
+
+- [ ] @AetherUnbound
```

Comment:
```suggestion
- [x] @AetherUnbound
```

Reply:
Oops, sorry for not doing this on my review!